throw new Error("error from main throw");
